eclass/tests/eapi8-dosym.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
```

Comment:
portable shebang pls ;)

Reply:
It's the same as in all other tests. And I hate these `env` shebangs (which assumes that `env` is installed in a particular location, so just shifts the problem).

Reply:
what if I run these tests on some old CentOS 6 Prefix installation?

Reply:
Or on NetBSD where bash is in /usr/pkg/bin?

Reply:
Yeah, sure. Continue like this and I'm going to drop tests altogether. :smile:

It's `/bin/bash` everywhere, and `tests-common.sh` sources `/lib/gentoo/functions.sh`. Does the latter exist on Prefix?

Reply:
we should make that relative then

Reply:
This is all part of a global issue that has little relation to this particular eclass and its tests.